manageiq-operator/config/crd/bases/manageiq.org_manageiqs.yaml

```diff
@@ -314,6 +314,11 @@ spec:
                   Image string used for the UI worker deployments
                   By default this is determined by the orchestrator pod
                 type: string
+              opentofuRunnerImage:
```

Comment:
Tests are failing because this is not sorted alphabetically

Reply:
@Fryguy Any thoughts on the naming?  Not sure if we prefer `opentofu`, `terraform` or something else.

Reply:
Thanks... Tests are passing now

Reply:
I think opentofuRunnerImage is good and it matches the class name on the other side...however would also like @agrare's thoughts

Reply:
Yeah I named the class `OpentofuRunner` on the ruby side so `opentofuRunnerImage` makes sense to me